nbiot.ts

```diff
@@ -279,10 +380,31 @@ namespace nbiot {
         // ignore empty lines
         if (lineEnd > 0) {
             lines.push(rxData.substr(0, lineEnd))
+            control.inBackground(checkResponseLine)
         }
         rxData = rxData.substr(lineEnd + 2)
     }
 
+    let receivedMessageLength = 0
+    function checkResponseLine() {
+        if (lines.length == 0) {
+            return
+        }
+        let lastLine = lines[lines.length - 1]
+        if (lastLine.indexOf("+NSONMI") == 0) {
+            // +NSONMI: <socket>,<length>
+            receivedMessageLength = parseInt(lastLine.substr(11))
```

Comment:
This will only work for socket <= 9 (single digit).  Which is probably fine.  But maybe split on comma instead?

Reply:
The EE-NBIOT-01 only has 7 sockets, but I agree split reads better.